.github/ISSUE_TEMPLATE/patternfly-issue-template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: PatternFly issue template
+about: Please populate the PatternFly issue template so we can learn more about your
+  needs.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Can we also add a question "is there a workaround?"

Reply:
Yes, and maybe we wrap that into @redallen 's question about it being blocking. What about "If this issue is a bug, Is this issue blocking you or is there a work around?" cc @dgutride